src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -504,24 +504,24 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
   @Getter(useStore) isRoleStaff!: boolean
 
   // Local variables
-  protected orgPerson: OrgPersonIF = null // current org/person being added/edited
-  protected orgPersonFormValid = true // model value for org/person form validity
-  protected inProgressMailingAddress: AddressIF = {}
-  protected inProgressDeliveryAddress: AddressIF = {}
```

Comment:
This gives a build error that `{} is not valid for AddressIF`, so changed the syntax.

I prefer the new syntax anyway since it keeps all variable initializations as "variable = value" (with type if needed).